### DISCUSSION TITLE
adaptive Random Forest not working

### DISCUSSION THREAD

**Discussion by divyajennifer** [online-ml/river#1341] [Q&A] [answered]
I had written used this below code with my dataset , but i get an error stating " AttributeError: module 'river.ensemble' has no attribute 'AdaptiveRandomForestClassifier'"

**model = ensemble.AdaptiveRandomForestClassifier(seed=8, leaf_prediction="mc")

metric = metrics.Accuracy()

evaluate.progressive_val_score(dataset, model, metric)**

**Comment by MaxHalford** [comment]
Hey there. What version of River are you using?

**Reply by divyajennifer** [reply]
using version 0.19.0

![image](https://github.com/online-ml/river/assets/148432994/bc189a5a-6b51-496c-b00a-dc8b575c6067)
Requirement already satisfied: river in /usr/local/lib/python3.10/dist-packages (0.19.0)
Requirement already satisfied: numpy>=1.25.2 in /usr/local/lib/python3.10/dist-packages (from river) (1.26.1)
Requirement already satisfied: scipy>=1.5 in /usr/local/lib/python3.10/dist-packages (from river) (1.11.3)
Requirement already satisfied: pandas>=1.3 in /usr/local/lib/python3.10/dist-packages (from river) (1.5.3)
Requirement already satisfied: python-dateutil>=2.8.1 in /usr/local/lib/python3.10/dist-packages (from pandas>=1.3->river) (2.8.2)
Requirement already satisfied: pytz>=2020.1 in /usr/local/lib/python3.10/dist-packages (from pandas>=1.3->river) (2023.3.post1)
Requirement already satisfied: six>=1.5 in /usr/local/lib/python3.10/dist-packages (from python-dateutil>=2.8.1->pandas>=1.3->river) (1.16.0)

**Reply by smastelini** [reply, marked as answer]
Hi @divyajennifer, ARF is now located in the forest module.

**Reply by divyajennifer** [reply]
ok Thanks for the update !!
